rdbms/src/main/java/io/spine/server/storage/jdbc/query/InsertAggregateRecordQuery.java

```diff
@@ -70,13 +77,30 @@ protected PreparedStatement prepareStatement(ConnectionWrapper connection) {
             final Timestamp timestamp = getRecord().getTimestamp();
             statement.setLong(3, timestamp.getSeconds());
```

Comment:
Please add a `todo` to get rid of this indexes. They are "magic numbers" at the moment.

Reply:
Done.